iocp/data/patterns.ini

```diff
@@ -7,19 +7,19 @@ pattern:	\b(([a-z0-9\-]{2,}\[?\.\]?)+(abogado|ac|academy|accountants|active|acto
 defang:		True
 
 [IP]
-pattern:	\b(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\b
+pattern:	\b((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$\b
 
 [Email]
-pattern:	\b([a-z][_a-z0-9-.]+@[a-z0-9-]+\.[a-z]+)\b
+pattern:	\b([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$\b
```

Comment:
Good idea, maybe it's better to use the W3C Email Regex used in HTML5 email field: 
\b[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)\b